packages/styles/accordion.css

```diff
@@ -3,67 +3,74 @@
   --accordion-trigger-background-color-expanded: var(--white);
   --accordion-trigger-text-color: var(--gray-90);
   --accordion-trigger-text-color-hover: var(--accent-primary);
-  --accordion-trigger-box-shadow-hover: var(--accent-primary);
+  --accordion-trigger-box-shadow-expanded: var(--accent-primary);
   --accordion-trigger-border-color: var(--gray-40);
   --accordion-trigger-icon-color: var(--gray-90);
   --accordion-panel-background-color: var(--white);
-  --accordion-panel-text-color: var(--gray-90);
+  --accordion-panel-text-color: var(--gray-80);
   --accordion-panel-border-color: var(--gray-40);
   --accordion-box-shadow-color: rgba(0, 0, 0, 0.15);
 }
 
 .cauldron--theme-dark {
   --accordion-trigger-background-color: var(--accent-medium);
-  --accordion-trigger-background-color-expanded: var(--accent-dark);
-  --accordion-trigger-text-color: var(--white);
-  --accordion-trigger-text-color-hover: var(--accent-light);
-  --accordion-trigger-box-shadow-hover: var(--accent-info);
```

Comment:
This can't be removed without it being a breaking change.

Reply:
These variables are in 2 different scopes, `:root: ` and `.cauldron--theme-dark`. If someone was depending on that variable existing within the `:root` scope, it would no longer have a value.

Reply:
I realize Github UI was being funky and showing this as changed when it got added below, dismissed.